src/us/kbase/workspace/test/database/provenance/FundingReferenceTest.java

```diff
@@ -189,64 +158,48 @@ private void buildFundingRefFailWithError(final FundingReference.Builder builder
 	}
 
 	@Test
-	public void buildFailInvalidPIDs() throws Exception {
-		for (String invalidPid : INVALID_PID_LIST) {
-			buildFundingRefFailWithError(
-					FundingReference.getBuilder(FUNDER_NAME_STRING).withFunderID(invalidPid),
-					"Illegal format for funderID: \"" + invalidPid + "\"\n" +
-							"It should match the pattern \"^([a-zA-Z0-9][a-zA-Z0-9\\.]+)\\s*:\\s*(\\S.+)$\"");
-		}
-	}
-
-	@Test
-	public void buildFailInvalidAwardURL() throws Exception {
+	public void buildFailInvalidGrantURL() throws Exception {
 		buildFundingRefFailWithError(
-				FundingReference.getBuilder(FUNDER_NAME_STRING).withAwardURL(FUNDER_ID_STRING),
-				"Illegal awardURL url '" + FUNDER_ID_STRING + "': unknown protocol: ror");
+				FundingReference.getBuilder(ORG_1).withGrantURL(GRANT_ID_STRING),
+				"Illegal grantURL url '" + GRANT_ID_STRING + "': no protocol: " + GRANT_ID_STRING);
 
 		buildFundingRefFailWithError(
-				FundingReference.getBuilder(FUNDER_NAME_STRING)
-						.withAwardURL("a random string with no protocol"),
-				"Illegal awardURL url 'a random string with no protocol': no protocol: a random string with no protocol");
+				FundingReference.getBuilder(ORG_1)
+						.withGrantURL("a random string with no protocol"),
+				"Illegal grantURL url 'a random string with no protocol': no protocol: a random string with no protocol");
 
 		buildFundingRefFailWithError(
-				FundingReference.getBuilder(FUNDER_NAME_STRING).withAwardURL("https://kb^ase.us/"),
-				"Illegal awardURL url 'https://kb^ase.us/': Illegal character in authority at " +
+				FundingReference.getBuilder(ORG_1).withGrantURL("https://kb^ase.us/"),
+				"Illegal grantURL url 'https://kb^ase.us/': Illegal character in authority at " +
 						"index 8: https://kb^ase.us/");
 
 		// class input
 		buildFundingRefFailWithError(
-				FundingReference.getBuilder(FUNDER_NAME_STRING)
-						.withAwardURL(new URL("https://kb^ase.us/")),
-				"Illegal awardURL url 'https://kb^ase.us/': Illegal character in authority at " +
+				FundingReference.getBuilder(ORG_1)
+						.withGrantURL(new URL("https://kb^ase.us/")),
+				"Illegal grantURL url 'https://kb^ase.us/': Illegal character in authority at " +
 						"index 8: https://kb^ase.us/");
 	}
 
 	@Test
-	public void buildFailNullOrWhitespaceFunderName() throws Exception {
-		for (String nullOrWs : WHITESPACE_STRINGS_WITH_NULL) {
-			buildFundingRefFailWithError(
-					FundingReference.getBuilder(nullOrWs),
-					"funderName cannot be null or whitespace only");
-		}
+	public void buildFailNullFunder() throws Exception {
+		buildFundingRefFailWithError(
+				FundingReference.getBuilder(null),
+				"funder cannot be null");
+
+		buildFundingRefFailWithError(
```

Comment:
This test is effectively no different from the one above since there's only one `getBuilder` method

Reply:
Removed redundant test.

Reply:
:+1: